service_api/schemas.py

```diff
@@ -1,6 +1,23 @@
 from marshmallow import Schema, fields, post_load, validate
 from service_api.models import State, City, OperationType, Realty, \
     RealtyDetails, RealtyType
+from service_api.grabbing_api.constants import ADDITIONAL_FILTERS
```

Comment:
why is it so long path?

Reply:
it is as long as the previous)
Should I change it?

Reply:
Can you?